### DISCUSSION TITLE
update MOOSE

### DISCUSSION THREAD

**Discussion by YNianer** [idaholab/moose#25562] [Q&A General]
The following issues occurred after updating moose compilation：

/Users/XXX/Desktop/VScode/moose/framework/src/executioners/Transient.C:739:27: error: no member named 'l2_norm_diff' in 'libMesh::NumericVector<double>'
  return current_solution.l2_norm_diff(old_solution) / current_solution.l2_norm();
         ~~~~~~~~~~~~~~~~ ^

**Comment by GiudGiud** [comment]
Hello

you need to update libmesh after updating moose

if you used the mamba build it may just be `mamba update —all`

If it doesn’t work, please see the pinned discussions post

Guillaume
